.github/workflows/test.yml

```diff
@@ -66,27 +66,28 @@ jobs:
   BrowserServicesKit:
     name: BSK unit tests
 
-    runs-on: macos-14
+    runs-on: macos-15
     timeout-minutes: 30
 
     steps:
       - name: Check out the code
         uses: actions/checkout@v4
         with:
           repository: 'duckduckgo/apple-browsers'
+          ref: 'sam/create-sharedpackages-directory' # TODO: For testing purposes, needs to be removed before merging
```

Comment:
This line will be deleted once the PR is approved and the upstream changes are merged